api/spec/scanner-adapter-openapi-v1.2.yaml

```diff
@@ -0,0 +1,568 @@
+          format: uri
+          example:
+            - https://security-tracker.debian.org/tracker/CVE-2017-8283
+        preferred_cvss:
+          $ref: "#/components/schemas/CVSSDetails"
+        cwe_ids:
+          type: array
+          items:
+            type: string
+          example: 
+            - CWE-476
+          description: |
+            The Common Weakness Enumeration Identifiers associated with this vulnerability.
+        vendor_attributes:
+          type: object
+          additionalProperties: true
+    Severity:
+      type: string
+      description: |
+        A standard scale for measuring the severity of a vulnerability.
+
+        * `Unknown` - either a security problem that has not been assigned to a priority yet or a priority that the
+          scanner did not recognize.
+        * `Negligible` - technically a security problem, but is only theoretical in nature, requires a very special
+          situation, has almost no install base, or does no real damage.
+        * `Low` - a security problem, but is hard to exploit due to environment, requires a user-assisted attack,
+          a small install base, or does very little damage.
+        * `Medium` - a real security problem, and is exploitable for many people. Includes network daemon denial of
+          service attacks, cross-site scripting, and gaining user privileges.
+        * `High` - a real problem, exploitable for many people in a default installation. Includes serious remote denial
+          of service, local root privilege escalations, or data loss.
+        * `Critical` - a world-burning problem, exploitable for nearly all people in a default installation. Includes
+          remote root privilege escalations, or massive data loss.
+      example: Low
+      enum:
+        - Unknown
+        - Negligible
+        - Low
+        - Medium
+        - High
+        - Critical
+    ErrorResponse:
+      type: object
+      properties:
+        error:
+          $ref: "#/components/schemas/Error"
+    Error:
+      type: object
+      properties:
+        message:
+          type: string
+          example: "Some unexpected error"
+    CVSSDetails:
+      type: object
+      properties:
+        score_v3:
+          type: number
+          format: float
+          description: |
+            The CVSS 3.0 score for the vulnerability.
+          example: 3.2
+        score_v2:
+          type: number
+          format: float
+          description: |
+            The CVSS 2.0 score for the vulnerability.
+        vector_v3:
+          type: string
+          description: |
+            The CVSS 3.0 vector for the vulnerability. 
+          example: CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N
+        vector_v2:
+          type: string
+          description: |
+            The CVSS 2.0 vector for the vulnerability. The string is of the form AV:L/AC:M/Au:N/C:P/I:N/A:N
+          example: AV:N/AC:L/Au:N/C:N/I:N/A:P
+    SBOMRequest:
```

Comment:
Can't we specify the SBOM format, such as SPDX and CycloneDX? Can the scanner return any formats?

Reply:
The `media_type` is introduced in the request, currently only  `application/spdx+json` and `application/vnd.cyclonedx+json` are supported, by default is `application/spdx+json`.

Reply:
Thanks for updating.